app/models/similar_occupation_standards.rb

```diff
@@ -8,6 +10,27 @@ def initialize(occupation_standard)
   end
 
   def top_five
-    OccupationStandard.search
+    OccupationStandard.__elasticsearch__.search(query).records
+  end
+
+  private
+
+  def query
+    {
+      size: 5,
+      query: {
+        more_like_this: {
```

Comment:
Is this just a custom name you give the query?

Reply:
No, this is a Elasticsearch specialized query: https://www.elastic.co/guide/en/elasticsearch/reference/7.17/query-dsl-mlt-query.html#query-dsl-mlt-query

Reply:
Does this method ensure that it never returns the initial input record? I skimmed the docs, but didn't see it explicitly mentioned, but would assume so.

Reply:
Yes it does, and I have a spec covering this requirement